cloud_governance/cost_expenditure/generate_cost_explorer_report.py

```diff
@@ -8,15 +8,18 @@
 
 class GenerateCostExplorerReport:
 
-    def __init__(self, cost_tags: list, es_host: str = '', es_port: str = '', es_index: str = '', metric_type: str = 'BlendedCost', file_name: str = ''):
+    def __init__(self, cost_tags: list, granularity: str, es_host: str = '', es_port: str = '', es_index: str = '', cost_metric: str = 'UNBLENDED_COST', file_name: str = '', start_date: str = '', end_date: str = ''):
+        self.start_date = start_date
+        self.end_date = end_date
+        self.granularity = granularity if granularity else 'DAILY'
+        self.cost_metric = cost_metric if cost_metric else 'UnblendedCost'
         self.cost_tags = cost_tags
```

Comment:
Delete it, should get it with default parameters:
 granularity: str='DAILY', cost_metric: str='UnblendedCost'